src/main/java/org/jcodec/movtool/Flatten.java

```diff
@@ -272,4 +273,4 @@ public void flatten(Movie movie, File video) throws IOException {
                 out.close();
```

Comment:
I think we are already closing the file right here, so that should be sufficient.